glide.yaml

```diff
@@ -4,7 +4,11 @@ import:
   version: master
 - package: github.com/giantswarm/micrologger
   version: master
+- package: github.com/giantswarm/microstorage
+  version: master
 - package: github.com/go-kit/kit
   version: v0.5.0
 - package: github.com/hashicorp/vault
   version: ~0.7.3
+- package: k8s.io/client-go
+  version: v3.0.0
```

Comment:
Note that I pinned the new dependencies here. Got some vendor weirdness in `kubernetesd`. I hope this helps.